src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -294,7 +312,6 @@ private function compileDynamicRoutes(RouteCollection $collection, bool $support
             'mark' => 0,
             'markTail' => 0,
             'supportsRedirections' => $supportsRedirections,
```

Comment:
I suggest to set `supportsRedirections` as a private property on the PhpMatcherDumper. Then we don't need to forward the variable to all the methods as an argument.

Reply:
done